openedx_events/apps.py

```diff
@@ -20,6 +23,10 @@ def general_signal_handler(sender, signal, **kwargs):  # pylint: disable=unused-
     #        "topic_a": { "event_key_field": "my.key.field", "enabled": True },
     #        "topic_b": { "event_key_field": "my.key.field", "enabled": False }
     # }"
+    if SIGNAL_PROCESSED_FROM_EVENT_BUS in kwargs and kwargs[SIGNAL_PROCESSED_FROM_EVENT_BUS] is True:
```

Comment:
Tests are failing because of coverage. Maybe we can test that signals are not sent when the SIGNAL_PROCESSED_FROM_EVENT_BUS is passed?